### ISSUE TITLE
individual object alerts redirection

### ISSUE THREAD

**Issue by emilleishida** [astrolabsoftware/fink-science-portal#399]
It would be nice to have the possibility of receiving an email or a slack message or some other type of notification everytime there is a new alert associated to a particular object I am interested in.

**Comment by JulienPeloton** [comment]
I agree this would be a nice feature. Although, we do not have the infrastructure to handle notifications yet. I'll keep you posted!
